tests/graphql/add-field-many-list.graphql

```diff
@@ -13,19 +13,24 @@ type Query {
   u: U @http(baseURL: "http://jsonplaceholder.typicode.com", path: "/us/1")
 }
 
-type U {
-  b: A @inline(path: ["a", "b"])
-  c: A @inline(path: ["a", "c"])
-  d: A @inline(path: ["a", "d"])
```

Comment:
Create new tests for `@field ` and don't modify these for inline. We will replace inline in a separate PR.